core/Command/Status.php

```diff
@@ -47,17 +48,40 @@ protected function configure() {
 
 		$this
 			->setDescription('show some status information')
-		;
+			->addOption(
+				'exit-code',
+				'e',
+				InputOption::VALUE_NONE,
+				'exit with 0 if running in normal mode, 1 when in maintenance mode, 2 when `./occ upgrade` is needed. Does not write any output to STDOUT.'
+			);
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
+
+		$maintenanceMode = $this->config->getSystemValueBool('maintenance', false);
+		$needUpgrade = Util::needUpgrade();
+
+		if ($input->getOption('exit-code')) {
+
+			if ($maintenanceMode === true) {
+			# checks maintenance mode first
+				return 1;
+
```

Comment:
```suggestion
			if ($maintenanceMode === true) {
				/* Checks maintenance mode first */
				return 1;

```
And same change for the 2 other comments.

Reply:
I've push a new commit that fixes the indentation of all three comments.